src/templates/rest-api/tests/test_server.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2022, Ansys Inc. Unauthorised use, distribution or duplication is prohibited
+
+import json
+import requests
+import random
+import multiprocessing
+from src.app import create_app
+
+
+class TestServer:
+    def setup_class(self):
```

Comment:
setup and teardown should be pytest fixtures. I can make them this if you need?

Reply:
Obviously, I tried to use fixtures but doing multiprocessing in a fixture is not so obvious. Therefore I roll back to a simpler approach. If you have any idea feel free to push a recommendation that works

Reply:
we really should use fixtures and avoid `multiprocessing` in tests if possible.

Reply:
why can you elaborate, please? What is your concern about multiprocessing?

Reply:
I dislike multiprocessing because it has burned me in the past but I'm not THAT set on this. If you're sure then we can leave as is.